docker/azure-function/release.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
add this to the `dev/release_docker_images.sh` file so we don't forget to release its image

Reply:
would be nice to rename folder name here as well :P